admin_guide/cluster_metrics.adoc

```diff
@@ -16,298 +16,382 @@ As an OpenShift administrator, you may want to view the metrics from all
-create their own.
+== Deploying the Metric Components
 
-The following is the definition of the *Grafana* service and replication
-controller:
+Since deploying and configuring all the metric components is handled by the metrics deployer, its is fairly
+simple to deploy everything in one step.
 
-====
-[source,yaml]
+The following examples show how to deploy metrics with and without persistent storage using the default 
+template parameters. If you wish, you can also specify any of the template parameters when calling these
+commands.
+
+=== Deploying with Persistent Storage
+
+The following command will set the Hawkular Metrics route to use `hawkular-metrics.example.com` and will
+be deployed using persistent storage.
+
+In order for this command to work, you must have a persistent volume of sufficient size available to be used.
+
+[options="nowrap"]
 ----
-apiVersion: "v1"
-kind: "List"
-items:
-  -
-    apiVersion: "v1"
-    kind: "Service"
-    metadata:
-      labels:
-        provider: "fabric8"
-        component: "grafana"
-      name: "grafana"
-    spec:
-      ports:
-        -
-          port: 80
-          targetPort: "http"
-      selector:
-        provider: "fabric8"
-        component: "grafana"
-  -
-    apiVersion: "v1"
-    kind: "ReplicationController"
-    metadata:
-      labels:
-        provider: "fabric8"
-        component: "grafana"
-      name: "grafana"
-    spec:
-      replicas: 1
-      selector:
-        provider: "fabric8"
-        component: "grafana"
-      template:
-        metadata:
-          labels:
-            provider: "fabric8"
-            component: "grafana"
-        spec:
-          containers:
-            -
-              env:
-                -
-                  name: "INFLUXDB_SERVICE_NAME"
-                  value: "INFLUXDB_MONITORING"
-                -
-                  name: "GRAFANA_DEFAULT_DASHBOARD"
-                  value: "/dashboard/file/kubernetes.json"
-              image: "fabric8/grafana:1.9.1_2"
-              name: "grafana"
-              ports:
-                -
-                  containerPort: 3000
-                  name: "http"
+$ oc process -f metrics.yaml -v \
+HAWKULAR_METRICS_HOSTNAME=hawkular-metrics.example.com \
+| oc create -f -
 ----
-====
 
-Save it to a file and create it with:
+=== Deploying without Persistent Storage
 
+The following command will set the Hawkular Metrics route to use `hawkular-metrics.example.com` and will
+be deployed without persistent storage.
+
+[IMPORTANT]
+====
+Since this is being deployed without persistent storage, metric data loss can occur.
+====
+
+[options="nowrap"]
 ----
-$ oc create -f <path/to/grafana.yaml>
+$ oc process -f metrics.yaml -v \
+HAWKULAR_METRICS_HOSTNAME=hawkular-metrics.example.com,USE_PERSISTENT_STORAGE=false \
+| oc create -f -
 ----
+
+== Accessing Metrics in the Console
+
+With the following setup, you will be able to view and access metrics from within the OpenShift console.
```

Comment:
It's probably enough to say that metrics are available from a pod's page under the `Browse` tab in the web console. We've been careful about how many screenshots we add to the doc because the UI changes often.